kedro/template/{{ cookiecutter.repo_name }}/src/environment.yml

```diff
@@ -0,0 +1,3 @@
+#pandas=0.24.1
```

Comment:
Do you mind removing this file? I suppose people who are interested in using `conda` can add it themselves, since you've documented it already in the docs for new project.